docs/editors/overview.md

```diff
@@ -418,6 +427,12 @@ Language Server Protocol (LSP). These extensions are not necessary for Metals to
 function but they improve the user experience. To learn more about Metals
 extensions, see [integrating a new editor](../integrations/new-editor.md).
 
+## Implicit decorations 
+
+**✅**: Additional information inferred from the code can be show within the code using additional virtual text.
```

Comment:
```suggestion
**✅**: Additional information inferred from the code can be show within the code using virtual text.
```
Super nitpicky, I think we can just remove the additional here.

Reply:
Fixed!